modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json

```diff
@@ -840,143 +840,143 @@
     },
     {
       "id": "91_mammals.gerp_conservation_score",
       "species": "pongo_abelii",
       "assembly": "Susie_PABv2",
       "type": "remote",
-      "filename_template" : "ftp://ftp.ensembl.org/pub/release-105/compara/conservation_scores/91_mammals.gerp_conservation_score/gerp_conservation_scores.pongo_abelii.Susie_PABv2.bw",
+      "filename_template" : "ftp://ftp.ensembl.org/pub/release-106/compara/conservation_scores/91_mammals.gerp_conservation_score/gerp_conservation_scores.pongo_abelii.Susie_PABv2.bw",
       "annotation_type": "gerp"
     },
     {
       "id": "91_mammals.gerp_conservation_score",
       "species": "rattus_norvegicus",
       "assembly": "mRatBN7.2",
       "type": "remote",
-      "filename_template" : "ftp://ftp.ensembl.org/pub/release-105/compara/conservation_scores/91_mammals.gerp_conservation_score/gerp_conservation_scores.rattus_norvegicus.mRatBN7.2.bw",
+      "filename_template" : "ftp://ftp.ensembl.org/pub/release-106/compara/conservation_scores/91_mammals.gerp_conservation_score/gerp_conservation_scores.rattus_norvegicus.mRatBN7.2.bw",
       "annotation_type": "gerp"
     },
     {
       "id": "91_mammals.gerp_conservation_score",
       "species": "sus_scrofa",
       "assembly": "Sscrofa11.1",
       "type": "remote",
-      "filename_template" : "ftp://ftp.ensembl.org/pub/release-105/compara/conservation_scores/91_mammals.gerp_conservation_score/gerp_conservation_scores.sus_scrofa.Sscrofa11.1.bw",
+      "filename_template" : "ftp://ftp.ensembl.org/pub/release-106/compara/conservation_scores/91_mammals.gerp_conservation_score/gerp_conservation_scores.sus_scrofa.Sscrofa11.1.bw",
       "annotation_type": "gerp"
     },
     {
       "id": "58_amniotes.gerp_conservation_score",
       "species": "taeniopygia_guttata",
       "assembly": "bTaeGut1_v1.p",
       "type": "remote",
-      "filename_template" : "ftp://ftp.ensembl.org/pub/release-105/compara/conservation_scores/58_amniotes.gerp_conservation_score/gerp_conservation_scores.taeniopygia_guttata.bTaeGut1_v1.p.bw",
+      "filename_template" : "ftp://ftp.ensembl.org/pub/release-106/compara/conservation_scores/58_amniotes.gerp_conservation_score/gerp_conservation_scores.taeniopygia_guttata.bTaeGut1_v1.p.bw",
       "annotation_type": "gerp"
     },
     {
       "id": "65_fish.gerp_conservation_score",
       "species": "tetraodon_nigroviridis",
       "assembly": "TETRAODON8",
       "type": "remote",
-      "filename_template" : "ftp://ftp.ensembl.org/pub/release-105/compara/conservation_scores/65_fish.gerp_conservation_score/gerp_conservation_scores.tetraodon_nigroviridis.TETRAODON8.bw",
+      "filename_template" : "ftp://ftp.ensembl.org/pub/release-106/compara/conservation_scores/65_fish.gerp_conservation_score/gerp_conservation_scores.tetraodon_nigroviridis.TETRAODON8.bw",
       "annotation_type": "gerp"
     },
     {
       "id": "65_fish.gerp_conservation_score",
       "species": "salmo_salar",
       "assembly": "ICSASG_v2",
```

Comment:
The assembly also needs to be updated here.

Reply:
I meant the salmon assembly from ICSASG_v2 to Ssal_v3.1.

Reply:
Thanks I have updated the assembly for the GERP file.
I will do a separate PR for the removal of the Salmon VCF for the e106 release.

Reply:
Thanks!